src/features/donation/models/effects/campaign-ft-donation.ts

```
@@ -0,0 +1,243 @@
+                        maxFtStorageBalanceBig.minus(protocolFeeRecipientFtStorageBalanceBig),
+                        NATIVE_TOKEN_DECIMALS,
+                      ),
+                    },
+                  ]
+                : []),
+
+              /**
+               *? FT contract storage balance replenishment for the referrer account
+               */
+              ...(referrerStorageBalanceBig !== null &&
+              referrerStorageBalanceBig.lt(maxFtStorageBalanceBig)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)